package.json

```diff
@@ -23,7 +23,7 @@
     "url": "https://github.com/webex/webex-js-sdk"
   },
   "scripts": {
-    "clean": "rimraf \"packages/**/.coverage\" \"packages/**/dist\" \"packages/**/types\"",
+    "clean": "rimraf \"packages/@webex/*/.coverage\" \"packages/@webex/*/dist\" \"packages/@webex/*/types\"",
```

Comment:
Won't we have to do clean in the `packages/webex` folder too?

Reply:
We will get all the tooling folders and `calling` folder as well in `next` branch

Reply:
added packages/webex/dist and packages/webex/.coverage